monaco/yarn.lock

```diff
@@ -0,0 +1,3655 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@types/node@^13.1.8":
+  version "13.1.8"
+  resolved "https://artifactory.palantir.build/artifactory/api/npm/all-npm/@types/node/-/node-13.1.8.tgz#1d590429fe8187a02707720ecf38a6fe46ce294b"
```

Comment:
Hi @gatesn,

Thank you for making this available. Massively appreciated.

I am quite keen take this pyodide + python language server + monaco for a spin. Unfortunately it appears your lock file is using a local npm repository, and when I delete your lock file I have been struggling to find a dependency combination that runs according to the readme :(.

Might you be in a position to update this pull request with the yarn.lock file pointing to public repos?